### ISSUE TITLE
[card] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1282]
### Feedback

❌ Card should not take focus
- if selectable then something within the card (like a checkbox) should take focus… it is fine to draw the focus on the card though
- Need to follow the Card pattern. Suggest looking at https://inclusive-components.design/cards/

General Comments:
- Source Order. Suggest the Heading to come before the image (use flexbox to switch)
- Heading should be an html heading by default. If someone doesn’t want that behaviour they should need to override that by sending standard text into that slot

### Path forward

1. Remove `tabindex` on root
2. It's not immediately clear how we're all that far from the demos in the pattern link supplied. Will review in more depth.
3. Swap image/heading order in DOM
4. This is the current Spectrum CSS direction. If this is incorrect, we should make sure that they're also education consumers of the accessible construction of this pattern.
   - look at inverting the override order of the slots for more accessible headlines by default

Card is generally tricky, and there is a lot of talk a different levels of the Spectrum pipeline of revisiting it whole sale...not that this should prevent us from getting this version to be more accessible, but it may put scope on that work, so will search for more info on those plans as well.

cc @jnurthen

**Comment by hunterloftis** [comment]
grouping 1-3 for 1.0, follow up on 4

**Comment by Westbrook** [comment]
@jnurthen when you have a minute could you check in on the tab order that we're testing via https://card-like-anchor--spectrum-web-components.netlify.app/storybook/index.html?path=/story/card--href? I think this aligns with what you were asking, but we're also looking at adding "card links" and having your insight in making that a quality screen reader experience would be super helpful.
